src/main/java/org/prebid/server/bidder/pubmatic/PubmaticBidder.java

```diff
@@ -299,23 +320,27 @@ private HttpRequest<BidRequest> makeHttpRequest(BidRequest request) {
     private BidRequest modifyBidRequest(BidRequest request,
                                         List<Imp> imps,
                                         String publisherId,
-                                        PubmaticWrapper wrapper) {
+                                        PubmaticWrapper wrapper,
+                                        List<String> acat) {
 
         return request.toBuilder()
                 .imp(imps)
                 .app(modifyApp(request.getApp(), publisherId))
                 .site(modifySite(request.getSite(), publisherId))
-                .ext(resolveRequestExt(request, wrapper))
+                .ext(resolveRequestExt(wrapper, acat))
```

Comment:
As you can see [here](https://github.com/prebid/prebid-server/pull/2201/files#diff-97031dc8e001ea1f77649fc3643e2f0b7cc12501c06762d793f58475c5ae6af5R124): if new `Req.Ext` is empty, than we don't replace the old one.